item-impl/src/main/java/com/example/auction/item/impl/PItemState.java

```diff
@@ -54,4 +54,19 @@ public PItemStatus getStatus() {
         return item.map(PItem::getStatus).orElse(PItemStatus.NOT_CREATED);
     }
 
```

Comment:
With the addition of `OptionalSerializer`, this whole class may now be unnecessary.